docker-multinode/README.md

```diff
@@ -17,7 +16,7 @@ interested in just starting to explore Kubernetes, we recommend that you start t
 
 ## Prerequisites
 
-The only thing you need is a machine with **Docker 1.7.1 or higher**
+The only thing you need is a machine with **Docker 1.8.3 or higher**
```

Comment:
Kubernetes officially supports 1.8.3-1.10.3 with release 1.2.
Sorry for CentOS 6, but it is really worth it to not upgrade?